lib/ansible/playbook/collectionsearch.py

```diff
@@ -41,4 +46,14 @@ def _load_collections(self, attr, ds):
         if not ds:  # don't return an empty collection list, just return None
             return None
 
+        # This duplicates static attr checking logic from post_validate()
+        # because if the user attempts to template a collection name, it will
+        # error before it ever gets to the post_validate() warning.
+        env = Environment()
+        for collection_name in ds:
+            if is_template(collection_name, env):
+                display.warning('"collections" is not templatable, but we found: %s, '
+                                'it will not be templated and will be used "as is".' % (collection_name))
+
```

Comment:
This makes the linter explode:
```suggestion
```
_(https://app.shippable.com/github/ansible/ansible/runs/163982/1/tests)_

Reply:
Not sure how that made it in, but fixed in my local repo. Will push when we sort out the test.

Reply:
Fixed